src/ledger.js

```diff
@@ -991,6 +1015,12 @@ export class LedgerSignMultisigTransaction extends LedgerBitcoinInteraction {
     return (transactionSignature || []).map(inputSignature => `${signatureNoSighashType(inputSignature)}01`);
   }
 
+  hexSignaturesToBuffers(transactionSignature) {
+    // Ledger signatures include the SIGHASH byte (0x01) if signing for P2SH-P2WSH or P2WSH ...
+    // but NOT for P2SH ... This function should always return the signature with SIGHASH byte appended.
+    return (transactionSignature || []).map(inputSignature => Buffer.from(`${signatureNoSighashType(inputSignature)}01`,"hex"));
```

Comment:
Seems like maybe we ought to DRY this up?

https://github.com/unchained-capital/unchained-wallets/blob/8a9e07f98545ed5b9f084b9f2adbad70e0a1da39/src/ledger.js#L989-L991

Reply:
Sure